docs/source/10min_dask_cudf.ipynb

```diff
@@ -0,0 +1,2565 @@
+    "def add_ten(num):\n",
+    "    return num + 10\n",
+    "\n",
+    "print(df['a'].applymap(add_ten))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Histogramming"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Counting the number of occurrences of each unique value of variable."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0    1\n",
+      "1    1\n",
+      "2    1\n",
+      "3    1\n",
+      "4    1\n",
+      "5    1\n",
+      "6    1\n",
+      "7    1\n",
+      "8    1\n",
+      "9    1\n",
+      "[10 more rows]\n",
+      "dtype: int64\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(df.a.value_counts())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0    1\n",
+      "1    1\n",
+      "2    1\n",
+      "3    1\n",
+      "4    1\n",
+      "5    1\n",
+      "6    1\n",
+      "7    1\n",
+      "8    1\n",
+      "9    1\n",
+      "[10 more rows]\n",
+      "dtype: int64\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(ddf.a.value_counts().compute())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## String Methods"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Like pandas, cuDF provides string processing methods in the `str` attribute of `Series`. Full documentation of string methods is a work in progress. Please see the cuDF and nvstrings API documentation for more information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0       a\n",
+      "1       b\n",
+      "2       c\n",
+      "3    aaba\n",
+      "4    baca\n",
+      "5    None\n",
+      "6    caba\n",
+      "7     dog\n",
+      "8     cat\n",
+      "dtype: object\n"
+     ]
+    }
+   ],
+   "source": [
+    "s = cudf.Series(['A', 'B', 'C', 'Aaba', 'Baca', None, 'CABA', 'dog', 'cat'])\n",
+    "print(s.str.lower())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0       a\n",
+      "1       b\n",
+      "2       c\n",
+      "3    aaba\n",
+      "4    baca\n",
+      "5    None\n",
+      "6    caba\n",
+      "7     dog\n",
+      "8     cat\n",
+      "dtype: object\n"
+     ]
+    }
+   ],
+   "source": [
+    "ds = dask_cudf.from_cudf(s, npartitions=2)\n",
+    "print(ds.str.lower().compute())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Merge\n",
```

Comment:
Missing content?

Reply:
This is a weird one. Merge is the top level category for concat/join/append, but doesn't appear like it in the notebook.

```
Merge
------------

## Concat
```
This looks weird here, but is important for formatting in the actual docs.